### PR TITLE
[WIP]Sesh User and Sesh Organisations

### PR DESCRIPTION
**Pull request by upatricck**
Added an interface to use when adding and removing sesh users of a specific organisation.

The user to access the interface should be an admin of an organisation.

### CODE REVIEW

File: seshdash/views.py [hunk 1]
```diff
@@ -1129,3 +1138,80 @@ def user_notifications(request):
...
+        if request.method == 'POST':
+            form = SeshUserForm(request.POST, instance=user)
+            if form.is_valid():
+                form.save()
+                return redirect('manage_org_users')   
+            else:
+                print "The form is not valid: %s" % form.errors
+
+        user_sites = _get_user_sites(request)
+        print "The user sites are: %s" % user_sites
```
Comment (jlark) [inline seshdash/views.py]: can you replace with log statment??

File: seshdash/views.py [hunk 2]
```diff
@@ -1129,3 +1138,80 @@ def user_notifications(request):
...
+        user = Sesh_User.objects.filter(id=user_id).first()
+        form = SeshUserForm(instance=user)
+
+        if request.method == 'POST':
+            form = SeshUserForm(request.POST, instance=user)
+            if form.is_valid():
+                form.save()
+                return redirect('manage_org_users')   
+            else:
+                print "The form is not valid: %s" % form.errors
```
Comment (jlark) [inline seshdash/views.py]: log statement please

**Comment by upatricck** [comment]
Anything else we can add to this request?

**Comment by upatricck** [comment]
Screen shots
screenshot from 2016-10-11 16 23 32
screenshot from 2016-10-11 16 22 14
screenshot from 2016-10-11 16 21 54

**Pull request merged**